docs/api/actions-and-connectors/execute.asciidoc

```diff
@@ -50,6 +50,104 @@ depending on the connector type. For information about the parameter properties,
 refer to <<action-types>>.
 +
 --
+.Jira connectors
+[%collapsible%open]
+====
+`subAction`::
+(Required, string) The action to test. Valid values include: `fieldsByIssueType`,
+`getFields`, `getIncident`, `issue`, `issues`, `issueTypes`, and `pushToService`.
+
+`subActionParams`::
```

Comment:
The `getFields` and `issueTypes` are missing. Is that intended?

Reply:
Per above, I omitted further details since they didn't seem to require `subActionParams` in my tests.

Reply:
I thought that the `subActionParams` were required. Thanks!